src/platform/test/functional/apps/dashboard_elements/input_control_vis/input_control_options.ts

```diff
@@ -83,6 +82,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
       it('should replace existing filter pill(s) when new item is selected', async () => {
         await comboBox.clear('listControlSelect0');
+        await common.sleep(500); // give time for filter to be removed and event handlers to fire
```

Comment:
Is there a way to do this without a sleep? Like maybe checking that the control no longer has a selected value?

Reply:
Good idea. Added in 7677abb.